app/code/Magento/Customer/Test/Unit/Model/Metadata/Form/FileTest.php

```diff
@@ -130,7 +130,7 @@ public function testExtractValueNoRequestScope($expected, $attributeCode = '', $
                 'entityTypeCode' => self::ENTITY_TYPE,
             ]
         );
-
+        $model->setRequestScope('');
```

Comment:
Could you explain why it's needed?

Reply:
Hello @ihor-sviziev ! 
Thanks for your review. This code adds PHP 8.1 compatibility for test and for usage strpos() function in [File.php](https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Customer/Model/Metadata/Form/File.php#L134)